YakushevY/HW7/3.py

```diff
@@ -0,0 +1,8 @@
+def calculates_number(string):
```

Comment:
see this implementation:
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result